### PR TITLE
Add list selector to TodoEditor(for #14)

### PR DESCRIPTION
**Pull request by AnubhaAgrawal**
Now, I think it's correct.

### CODE REVIEW

**Comment by untitaker** [comment]
Hello @AnubhaAgrawal, your editor is configured to use tabs instead of spaces, and it appears that this reformatted the entire file. This makes review very hard because the diff becomes very large.

If you run `tox -eflake8`, it should spit out a list of formatting problems with your code. Please fix those problems.

**Comment by hobarrera** [comment]
Looks like your editor reformatted the entire code ([see here](https://github.com/pimutils/todoman/pull/131/files)).
Not only will this custom code format fail style checks, but it also makes reading the changes very complex. :(

File: todoman/ui.py [hunk 1]
```diff
@@ -28,15 +28,18 @@ def __init__(self, todo, lists, formatter):
...
+        self.current_list = todo.list
         self.todo = todo
         self.lists = lists
         self.formatter = formatter
         self.saved = EditState.none
         self._loop = None
 
         self._msg_text = urwid.Text('')
-
+        urwid.RadioButton(self.lists, "label", state='first True',
```
Comment (untitaker) [inline todoman/ui.py]: * You are creating a RadioButton and discard it. Instead, you need to add it to the `pile_items` list or `items`, depending on which position you want the list selector to be. In *which position of the list* you want to add the `RadioButton` also depends on where you want the list selector to be.
* You will need multiple RadioButtons, one for each list to select.
* The first argument has to be a list to which the RadioButton will append itself to. `self.lists` cannot be that list. It should be `items` or `pile_items`, but since that list is only created further down, you will need to move the entire block down.
* `on_state_change` has to be a function that sets `self.current_list` to the list the RadioButton stands for. Something like this:

      def change_current_list(new_list):
          self.current_list = new_list

      for _list in self.lists:
          RadioButton(..., on_state_change=change_current_list, user_data=_list)

File: todoman/ui.py [hunk 2]
```diff
@@ -220,7 +224,6 @@ def compact(self, todo):
         """
         Returns a brief representation of a task, suitable for displaying
         on-per-line.
-
```
Comment (untitaker) [inline todoman/ui.py]: Removing these lines is not necessary.

File: todoman/ui.py [hunk 3]
```diff
@@ -140,12 +143,13 @@ def _save(self, btn=None):
...
 
     def _save_inner(self):
+        self.todo.list = self.current_list
         self.todo.summary = self.summary
         self.todo.description = self.description
         self.todo.location = self.location
         self.todo.due = self.formatter.parse_date(self.due)
         self.todo.start = self.formatter.parse_date(self.dtstart)
-
+
```
Comment (untitaker) [inline todoman/ui.py]: This was probably your editor, please also undo this change (added whitespace)

File: todoman/ui.py [hunk 4]
```diff
@@ -140,12 +143,19 @@ def _save(self, btn=None):
             raise urwid.ExitMainLoop()
 
     def _save_inner(self):
+        self.todo.list = self.current_list
         self.todo.summary = self.summary
         self.todo.description = self.description
         self.todo.location = self.location
+<<<<<<< HEAD
```
Comment (untitaker) [inline todoman/ui.py]: It appears you tried to resolve the conflict, but you need to remove one of the two "versions" manually (otherwise git could do it automatically). Please remove these added lines.

File: todoman/ui.py [hunk 5]
```diff
@@ -140,12 +143,19 @@ def _save(self, btn=None):
...
+        self.todo.due = self.formatter.parse_date(self.due)
+        self.todo.start = self.formatter.parse_date(self.dtstart)
+        
+=======
         self.todo.due = self.formatter.parse_datetime(self.due)
         self.todo.start = self.formatter.parse_datetime(self.dtstart)
 
+>>>>>>> 1bcdd38a08f1daa2cac75d15c1b31b439eb06497
```
Comment (untitaker) [inline todoman/ui.py]: Same thing here, this is not valid Python :)

File: todoman/ui.py [hunk 6]
```diff
@@ -271,8 +279,12 @@ def _format_date(self, date):
         * if date == today or tomorrow: "Today" or "Tomorrow"
         * else: return a string representing that date
         * if no date is supplied, it returns empty_date
+<<<<<<< HEAD
```
Comment (untitaker) [inline todoman/ui.py]: Same thing here.

**Comment by AnubhaAgrawal** [comment]
Sorry for these commits. I commit on my repository but I cloned it from your repository so these commits will go to your repository.

**Comment by untitaker** [comment]
It's no problem, we will squash the entire commit history anyway.

**Comment by untitaker** [comment]
Continuing at #134
